rtp/mix.exs

```diff
@@ -0,0 +1,36 @@
+defmodule Membrane.Demo.RTP.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :membrane_demo_rtp,
+      version: "0.1.0",
+      elixir: "~> 1.9",
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      mod: {Membrane.Demo.RTP, []},
+      extra_applications: [:logger]
+    ]
+  end
+
+  # Run "mix help deps" to learn about dependencies.
+  defp deps do
+    [
+      # {:dep_from_hexpm, "~> 0.3.0"},
+      # {:dep_from_git, git: "https://github.com/elixir-lang/my_dep.git", tag: "0.1.0"}
+      {:membrane_core, [env: :prod, path: "~/membrane-core", override: true]},
+      {:membrane_bin_rtp, path: "~/membrane_bin_rtp"},
```

Comment:
It seems to be exactly the right direction, and while I'd happily provide more in-depth feedback; _this_ dependency isn't published so I can't run this.

Reply:
It is going to be published along with Membrane v0.5 which is happening this week.